engine/include/cubos/engine/audio/bridge.hpp

```diff
@@ -0,0 +1,35 @@
+/// @file
+/// @brief Class @ref cubos::engine::AudioBridge.
+/// @ingroup audio-plugin
+
+#pragma once
+
+#include <utility>
+
+#include <cubos/engine/assets/bridges/file.hpp>
+#include <cubos/engine/audio/audio.hpp>
+
+namespace cubos::engine
+{
+    /// @brief Bridge which loads and saves @ref Sound assets.
+    ///
+    /// Uses the format specified in @ref Audio::loadFrom and @ref Audio::writeTo
```

Comment:
I think you just copypasted this from somewhere else, as these methods don't exist. But yes, we should document the supported formats here (or a link to where they're available).